assets/src/edit-story/components/form/dateTime/index.js

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable header/header */
+/* Disable reason: This file is based on Gutenberg Datepicker */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { rgba } from 'polished';
+
+/**
+ * Internal dependencies
+ */
+import TimePicker from './timePicker';
+import DatePicker from './datePicker';
+
+const DateTimeWrapper = styled.div`
+  position: absolute;
+  top: 30px;
+  left: -255px;
+  box-shadow: 0 3px 30px rgba(25, 30, 35, 0.1);
+  border: 1px solid ${({ theme }) => rgba(theme.colors.bg.v0, 0.2)};
+  background-color: ${({ theme }) => theme.colors.fg.v1};
+  z-index: 1;
+  width: 270px;
+  padding: 4px;
+`;
+
+function DateTimePicker({ value, onChange, is12Hour = true, forwardedRef }) {
```

Comment:
Sometimes clicking inside this component on the background closes the overlay, sometimes not. I haven't really figured out when it happens, but it seems weird.